cdap-watchdog/src/main/java/io/cdap/cdap/metrics/store/DefaultMetricStore.java

```diff
@@ -87,6 +87,7 @@ public class DefaultMetricStore implements MetricStore {
   private static final String BY_COMPONENT = "component";
   private static final String BY_SCHEDULE = "schedule";
   private static final String BY_ONLY_COMPONENT = "only_component";
+  private static final String BY_ENTITY = "entity";
```

Comment:
Just wondering does it make more sense to use `BY_APP_ENTITY = "app_entity";`?

Reply:
Makes sense, done